prqlc/prqlc/build.rs

```diff
@@ -0,0 +1,14 @@
+use std::fmt::Error;
+use std::process::Command;
+// use vergen::EmitBuilder;
+
+pub fn main() -> Result<(), Error> {
+    let output = Command::new("git")
+        .args(["describe", "--tags"])
+        .output()
+        .unwrap();
```

Comment:
Doesn't this mean that the build will fail if the git command is not available?

Reply:
Yes, this will be a problem on some build platforms without git or if the build process strips git repo out of source code.